docs/cdn/_redirects

```diff
@@ -40,6 +40,8 @@ https://material-ui.dev/* https://material-ui.com/:splat 301!
 /lab/* /components/:splat 301
 /css-in-js/* /styles/:splat 301
 /premium-themes* https://themes.material-ui.com/ 301
+/api/* /components-api/:splat 301
```

Comment:
Since this redirects all `/api` paths, how would one use the new feature(add an API page)?
I tried to add an API page yesterday but kept on getting errors, now I see why.

Reply:
I haven't thought about it. What would be the use case for us? Maybe we would host it on a different domain?